codeworld-available-pkgs/codeworld-available-pkgs.cabal

```diff
@@ -10,317 +10,629 @@ Copyright:           (c) 2020, The CodeWorld Authors
-    integer-gmp,
-    integer-logarithms,
-    integration,
-    interpolation,
-    intervals,
-    invariant,
-    jsaddle,
-    kan-extensions,
-    keycode,
-    latex,
-    leancheck,
-    lens,
-    lifted-async,
-    lifted-base,
-    linear,
-    log-domain,
-    logict,
-    loop,
-    markov-chain,
-    massiv,
-    math-functions,
-    matrix,
-    megaparsec,
-    memory,
-    mersenne-random-pure64,
-    mfsolve,
-    microlens,
-    microlens-mtl,
-    miso,
-    mmorph,
-    monad-bayes,
-    monad-control,
-    -- monad-coroutine 0.9.1 broken for old base
-    monad-coroutine < 0.9.1,
-    monad-loops,
-    monad-parallel,
-    monad-unlift,
-    mono-traversable,
-    monoid-extras,
-    monoid-transformer,
-    monoidal-containers,
-    moo,
-    mtl,
-    mtl-compat,
-    mutable-containers,
-    mwc-probability,
-    mwc-random,
-    nats,
-    natural-transformation,
-    netwire,
-    network-uri,
-    newtype-generics,
-    non-negative,
-    numeric-prelude,
-    old-locale,
-    optics,
-    optics-core,
-    optics-extra,
-    optparse-applicative,
-    parallel,
-    parsec,
-    parser-combinators,
-    parsers,
-    patch,
-    pipes,
-    polysemy,
-    pqueue,
-    prelude-compat,
-    pretty,
-    pretty-simple,
-    pretty-show,
-    prettyprinter,
-    prim-uniq,
-    primitive,
-    probability,
-    profunctors,
-    protolude,
-    psqueues,
-    quickcheck-io,
-    random,
-    random-shuffle,
-    reactive-banana,
-    -- reanimate, (depends on chiphunk, which depends on native library)
-    reducers,
-    ref-tf,
-    reflection,
-    reflex,
-    reflex-dom,
-    reflex-dom-core,
-    regex-base,
-    regex-compat-tdfa,
-    regex-tdfa,
-    relude,
-    resourcet,
-    rio,
-    safe,
-    safe-exceptions,
-    scheduler,
-    scientific,
-    semialign,
-    semigroupoids,
-    semigroups,
-    servant,
-    set-cover,
-    singleton-bool,
-    -- singletons, (build error related to Int/Integer/Uniq)
-    smallcheck,
-    sop-core,
-    split,
-    splitmix,
-    statistics,
-    stm,
-    storable-complex,
-    storable-record,
-    streaming,
-    string-conversions,
-    svg-builder,
-    syb,
-    tagged,
-    tasty,
-    tasty-expected-failure,
-    tasty-hedgehog,
-    tasty-hunit,
-    template-haskell,
-    text,
-    tf-random,
-    tfp,
-    these,
-    time,
-    time-compat,
-    transformers,
-    transformers-base,
-    transformers-compat,
-    -- transient, (build fails: missing Semigroup instance for TransIO)
-    trifecta,
-    type-equality,
-    typelits-witnesses,
-    unagi-chan,
-    unbounded-delays,
-    unexceptionalio,
-    unicode,
-    unique,
-    unliftio,
-    unliftio-core,
-    unordered-containers,
-    utf8-string,
-    utility-ht,
-    uuid-types,
-    vault,
-    vector,
-    vector-algorithms,
-    vector-binary-instances,
-    vector-builder,
-    vector-space,
-    vinyl,
-    void,
-    witherable,
-    wl-pprint-annotated,
-    zenc
+  if impl(ghcjs -any)
```

Comment:
similar to https://github.com/google/codeworld/blob/e596c2092ff73a5ca97f77aafc706606c5d2db55/codeworld-base/codeworld-base.cabal#L41-L65